docs/admin/access.rst

```diff
@@ -90,15 +90,16 @@ Administration
     Includes all permissions available for the project.
 
 Review (only if :ref:`review workflow <reviews>` is turned on)
-    Can approve translations during review.
+    Can approve translations during the review.
```

Comment:
drop "the"

Reply:
You’re right! Can you add it as a suggestion, or should I write it in?